### PR TITLE
Add support to use MiniMessage format for inventory title

### PR DESCRIPTION
**Pull request by Akiranya**

### CODE REVIEW

File: src/main/java/me/hsgamer/bettergui/paperspec/PaperSpec.java [hunk 1]
```diff
@@ -1,35 +1,62 @@
...
-        ItemModifierBuilder.INSTANCE.register(PaperSkullModifier::new, "paper-skull", "paper-head");
-        ItemModifierBuilder.INSTANCE.register(AdventureNameModifier::new, "mini-name", "adventure-name", "name$");
-        ItemModifierBuilder.INSTANCE.register(AdventureLoreModifier::new, "mini-lore", "adventure-lore", "lore$");
+        ItemModifierBuilder.INSTANCE.register(PaperSkullModifier::new, "paper-skull", "paper-head", "skull$", "head$");
+        ItemModifierBuilder.INSTANCE.register(AdventureNameModifier::new, "mini-name", "name$");
+        ItemModifierBuilder.INSTANCE.register(AdventureLoreModifier::new, "mini-lore", "lore$");
+
+        InventoryBuilder.INSTANCE.register(pair -> {
+            Map<String, Object> map = pair.getValue();
+            String title = Optional.ofNullable(MapUtil.getIfFound(map, "mini-title", "title$"))
```
Comment (HSGamer) [inline src/main/java/me/hsgamer/bettergui/paperspec/PaperSpec.java]: I think you can just use `holder#getTitle` because it'll be Component anyways.

**Review by HSGamer — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 2]
```diff
@@ -11,7 +11,7 @@
   </parent>
 
   <artifactId>PaperSpec</artifactId>
-  <version>1.2-SNAPSHOT</version>
+  <version>1.3-SNAPSHOT</version>
```
Comment (HSGamer) [inline pom.xml]: You don't have to bump the version. I prepared this for the next release.

**Review by HSGamer — CHANGES_REQUESTED** [review]

File: src/main/java/me/hsgamer/bettergui/paperspec/PaperSpec.java (hunk 1 above)
Comment (Akiranya) [reply src/main/java/me/hsgamer/bettergui/paperspec/PaperSpec.java]: Nice catch! I am not familar enough with the API yet. Not sure if I correctly understand what you said, but I tried to implement it

**Comment by Akiranya** [comment]
Oh hold on for the merge, there's some tiny things to be done. Should be ready in 3 mins

File: src/main/java/me/hsgamer/bettergui/paperspec/PaperSpec.java (hunk 1 above)
Comment (HSGamer) [reply src/main/java/me/hsgamer/bettergui/paperspec/PaperSpec.java]: That's ok now.

**Comment by HSGamer** [comment]
I'm gonna merge this PR. Can you update the wiki on the new changes?

**Comment by HSGamer** [comment]
Oops, I didn't see your last message. Can you open another PR after you finish?

**Comment by Akiranya** [comment]
> Oops, I didn't see your last message. Can you open another PR after you finish?

Oh sure! A new PR is coming

**Comment by Akiranya** [comment]
> I'm gonna merge this PR. Can you update the wiki on the new changes?

Will do

**Comment by Akiranya** [comment]
Done!

**Pull request merged**